src/labels/CustomLabels.labels

```diff
@@ -5519,6 +5519,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Edit</shortDescription>
         <value>Edit</value>
     </labels>
+    <labels>
+        <fullName>commonEditPaymentInformation</fullName>
```

Comment:
@melissabarber Updated label here

Reply:
Made one suggestion. After that's committed, this looks good to go.